app/models/flag.rb

```diff
@@ -28,9 +28,9 @@ def self.valid_applies_to_types
   end
 
   def self.reset_flag_settings!
-    # Flags are memoized for better performance. After the update, we need to reload them in all processes.
+    # Flags are cached in Redis for better performance. After the update,
+    # we need to reload them in all processes.
     PostActionType.reload_types
```

Comment:
We don't need it anymore. It was used to communicate between unicorn processes. Since we are using Redis cache it is enough when one process expire it